### PR TITLE
[13.0][MIG] rma_sale: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT26283

Depends on:
- [x] https://github.com/OCA/rma/pull/177

### CODE REVIEW

File: rma_sale/__manifest__.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
 {
     "name": "Return Merchandise Authorization Management - Link with Sales",
     "summary": "Sale Order - Return Merchandise Authorization (RMA)",
-    "version": "12.0.1.5.2",
+    "version": "13.0.1.0.0",
     "development_status": "Beta",
```
Comment (pedrobaeza) [inline rma_sale/__manifest__.py]: This should be `Production/Stable`

**Review by pedrobaeza — APPROVED** [review]

File: rma_sale/__manifest__.py (hunk 1 above)
Comment (ernestotejeda) [reply rma_sale/__manifest__.py]: Sure!, I'll change it right now

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by chienandalu — APPROVED** [review]
Thanks!

**Pull request merged**